src/b64.ml

```diff
@@ -41,52 +44,50 @@ let decode ?alphabet input =
     | _ when input.[length - 1] = padding -> 1
     | _ -> 0
   in
-  let output = Bytes.make (words * 3 - padding) '\000' in
+  let output = Bytes.make ((words * 3) - padding) '\000' in
   for i = 0 to words - 1 do
-    let a = of_char ?alphabet (String.get input (4 * i + 0))
-    and b = of_char ?alphabet (String.get input (4 * i + 1))
-    and c = of_char ?alphabet (String.get input (4 * i + 2))
-    and d = of_char ?alphabet (String.get input (4 * i + 3)) in
+    let a = of_char ?alphabet input.[(4 * i) + 0]
+    and b = of_char ?alphabet input.[(4 * i) + 1]
+    and c = of_char ?alphabet input.[(4 * i) + 2]
+    and d = of_char ?alphabet input.[(4 * i) + 3] in
     let n = (a lsl 18) lor (b lsl 12) lor (c lsl 6) lor d in
     let x = (n lsr 16) land 255
     and y = (n lsr 8) land 255
     and z = n land 255 in
-    Bytes.set output (3 * i + 0) (char_of_int x);
+    Bytes.set output ((3 * i) + 0) (char_of_int x) ;
     if i <> words - 1 || padding < 2 then
-      Bytes.set output (3 * i + 1) (char_of_int y);
+      Bytes.set output ((3 * i) + 1) (char_of_int y) ;
     if i <> words - 1 || padding < 1 then
-      Bytes.set output (3 * i + 2) (char_of_int z);
-  done;
+      Bytes.set output ((3 * i) + 2) (char_of_int z)
+  done ;
   Bytes.unsafe_to_string output
 
 let decode_opt ?alphabet input =
-  try
-    Some (decode ?alphabet input)
-  with
-    Not_found -> None
+  try Some (decode ?alphabet input) with Not_found -> None
 
-let encode ?(pad=true) ?alphabet input =
+let encode ?(pad = true) ?alphabet input =
   let length = String.length input in
-  let words = (length + 2) / 3 in (* rounded up *)
+  let words = (length + 2) / 3 in
+  (* rounded up *)
```

Comment:
i disagree with this ocamlformat change, is there a way to configure it to not insert the newline?

Reply:
I did not find something about that in `ocamlformat` ... Should be an issue (@gpetiot?).

Reply:
You can place the comment just before the `in` keyword.